x-pack/plugin/core/src/test/java/org/elasticsearch/protocol/xpack/watcher/GetWatchResponseTests.java

```diff
@@ -126,15 +124,15 @@ private static BytesReference simpleWatch() {
 
     private static WatchStatus randomWatchStatus() {
         long version = randomLongBetween(-1, Long.MAX_VALUE);
-        WatchStatus.State state = new WatchStatus.State(randomBoolean(), nowWithMillisResolution());
+        WatchStatus.State state = new WatchStatus.State(randomBoolean(), DateUtils.nowWithMillisResolution());
```

Comment:
I would `import static` this method to have a cleaner/nicer code.

Reply:
I am not sure, I personally prefer to use a class name when context is not obvious. For the same reason I would use `ZonedDateTime.now` instead of `now()`